### ISSUE TITLE
Pagination component accessibility

### ISSUE THREAD

**Issue by joshikeerti** [rafaelpimpa/buefy#327]
Currently the pagination component uses <a> tags without href and is not accessible to keyboard users and screenreaders. Is there any way to add the aria role or other attributes to it to make it accessible?

**Buefy** version: 0.5
**Vuejs** version: 2.4
**OS/Browser**: MacOS Safari/ Chrome

**Comment by rafaelpimpa** [comment]
Close it with https://github.com/rafaelpimpa/buefy/commit/8f4fbff766e437e83f5328c70e90d957283daf2d and https://github.com/rafaelpimpa/buefy/commit/30c8027640ad0003c166079515241adfdd609467
